plugins/source/aws/client/testing.go

```diff
@@ -150,3 +153,27 @@ func validateNoEmptyColumnsExcept(t *testing.T, tables schema.Tables, messages m
 		}
 	}
 }
+
+func validateTagData(t *testing.T, tables schema.Tables, messages message.SyncMessages) {
+	tablesWithIssues := []string{}
+	for _, table := range tables.FlattenTables() {
+		index := table.Columns.Index("tags")
+		if index == -1 {
+			continue
+		}
+		records := messages.GetInserts().GetRecordsForTable(table)
+		for _, resource := range records {
+			arr := resource.Column(index)
+			for i := 0; i < arr.Len(); i++ {
+				val := arr.GetOneForMarshal(i).(json.RawMessage)
```

Comment:
Won't it panic on nil?

Reply:
It should never be null. If it was empty the test would have failed at this point